cadence/transactions/kittyItems/add_nft_images_for_new_kind.cdc

```diff
@@ -0,0 +1,27 @@
+import KittyItems from "../../contracts/KittyItems.cdc"
+ 
+ 
+transaction {
+
+  let minter: &KittyItems.NFTMinter
+
+  prepare(signer: AuthAccount) {
+
+    self.minter = signer.borrow<&KittyItems.NFTMinter>(from: KittyItems.MinterStoragePath)
```

Comment:
This is a redundant check since only the minter is allowed to call the `addNewImagesForKind` method on the minter. This is a 'fail fast' condition.